package.json

```diff
@@ -15,7 +15,8 @@
   "private": true,
   "dependencies": {
     "@uidotdev/usehooks": "^2.4.1",
-    "react-virtuoso": "^4.12.1"
+    "react-virtuoso": "^4.12.1",
+    "sass": "^1.81.0"
```

Comment:
Should be under devDependencies.
Yesterday morning we talked about using https://www.npmjs.com/package/sass-embedded is it a better option? @michaelpande @SjurSutterudSagen